migrations/20190129083947-add-collective-country_iso.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Collectives', 'countryISO', Sequelize.STRING).then(() => {
```

Comment:
It's a detail but a good practice to only use the size needed. As we know that ISO 3166 defines country codes as two characters long, we could go with `Sequelize.STRING(32) `